go/config/defaults/0-base-config.yaml

```diff
@@ -8,11 +8,11 @@ network:
   batch:
     interval: 1s
     maxInterval: 1s # if this is greater than batch.interval then we make batches more slowly when there are no transactions
-    maxSize: 45000 # around 45kb - around 200 transactions / batch
```

Comment:
there was a reason we picked batch size to be lower but i cant recall. The only thing i recall as a hard limit was 64kb for accepting eth transactions in the geth code